### PR TITLE
Fix Service Discovery failures with Globalnet jobs

### LINKED ISSUE #471: Service Discovery e2e tests in Globalnet jobs are failing in CI
**What happened**:
Service Discovery e2e tests in Globalnet jobs are failing in CI

### PR DESCRIPTION
**Pull request by sridhargaddam**
Recently Globalnet was updated to annotate only exported services and
Shipyard e2e framework was modified to export a service once its created.
This PR modifies the reference to shipyard to pull the latest codebase.

Fixes issue: https://github.com/submariner-io/lighthouse/issues/471
Signed-Off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

**Review by aswinsuryan — APPROVED** [review]

**Review by vthapar — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]

**Review by mkolesnik — APPROVED** [review]
Still seems to be failing though

**Comment by mangelajo** [comment]
@sridhargaddam the globalnet jobs seem to be failing for globalnet reasons yet. Can you check?

**Review by tpantelis — APPROVED** [review]

**Comment by sridhargaddam** [comment]
There are no postmortem logs, so retriggering the CI run.

**Comment by sridhargaddam** [comment]
@mangelajo @vthapar @mkolesnik  : Fixed the Globalnet failures by modifying LH e2e tests. But we need an updated subctl (as we use it for verify connectivity) which includes these changes. So we are in some kind of cyclic dependency :(

I manually built an updated subctl with these changes and verified the discovery tests. Can we bypass CI result and merge this PR? After this PR gets merged, the next steps would be to update references to Lighthouse in Submariner-operator repo so that we can get updated subctl.

```
Ran 5 of 34 Specs in 133.180 seconds
SUCCESS! -- 5 Passed | 0 Failed | 0 Pending | 29 Skipped
```

**Pull request merged**
